### PR TITLE
latlng input fix

### PR DESCRIPTION
**Pull request by alias01**

### CODE REVIEW

File: src/components/templates/TemplateInventory.vue [hunk 1]
```diff
@@ -280,19 +285,15 @@ export default {
             this.$emit('updateRequest', spotData);
         },
 
-        updateLat(spotData, lat) {
+        updateLatLng(spotData, latlng) {
+            const coordinate = latlng.split(',');
```
Comment (arrogant-ant) [inline src/components/templates/TemplateInventory.vue:289]: Create a util method for extracting lat lng from string and use the same in SRP to parse latlng for query param

Comment (alias01) [reply src/components/templates/TemplateInventory.vue]: done

File: src/includes/LatLng.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Take comma seperated latlng and convert into array.
+ * @param {string} latlng string.
+ * @return {Array} coordinate.
```
Comment (arrogant-ant) [inline src/includes/LatLng.js:4]: Add details like pos0 -> lat pos1-> Lng

File: src/includes/LatLng.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+ */
+function latLngFilter(latlng) {
+    let coordinate = latlng.split(',');
+    coordinate = coordinate.map((coord) => coord.trim());
+    return coordinate;
```
Comment (arrogant-ant) [inline src/includes/LatLng.js:9]: add some validation checks like after spilt length == 2

File: src/includes/LatLng.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Take comma seperated latlng and convert into array.
+ * @param {string} latlng string.
+ * @return {Array} coordinate.
+ */
+function latLngFilter(latlng) {
```
Comment (arrogant-ant) [inline src/includes/LatLng.js:6]: rename function as its not a filter maybe 'getCoordinate'

**Review by arrogant-ant — CHANGES_REQUESTED** [review]

File: src/includes/LatLng.js (hunk 2 above)
Comment (alias01) [reply src/includes/LatLng.js]: done

File: src/includes/LatLng.js (hunk 3 above)
Comment (alias01) [reply src/includes/LatLng.js]: done

File: src/includes/LatLng.js (hunk 4 above)
Comment (alias01) [reply src/includes/LatLng.js]: done

**Review by arrogant-ant — APPROVED** [review]

**Pull request merged**